.github/workflows/test.yml

```diff
@@ -257,187 +255,3 @@ jobs:
-          
-          }
-        env:
-          BUILD_TARGETS: ${{ join(matrix.config.rust-targets, ',') }}
-          RUST_TOOLCHAIN: ${{ matrix.config.rust-version }}
-
-      # Identical to the build process, but running more test commands
-      # ! ci-cargo requires '--' to be wrapped in quotes (passed as an explicit string)
-      - name: Run tests
-        run: |
-          . ./ci-cargo.ps1
-          $targets = $env:BUILD_TARGETS.Split(',')
-          $toolchain = $env:RUST_TOOLCHAIN
-          $itt = 0..($targets.Length - 1)
-          foreach($i in $itt) {
-            if ($targets[$i] -eq "") {
-              $target = 'default'
-            }
-            else {
-              $target = $targets[$i]
-            }
-
-            ci-cargo +$toolchain test --manifest-path extendr-engine/Cargo.toml --features tests-all $(if($target -ne 'default') {"--target=$target"} ) $env:EXTRA_ARGS '--' --nocapture -ActionName "Test extendr-engine \w tests-all for $target target"
-
-            ci-cargo +$toolchain test --manifest-path extendr-api/Cargo.toml --features tests $(if($target -ne 'default') {"--target=$target"} ) $env:EXTRA_ARGS '--' --nocapture -ActionName "Test extendr-api \w tests for $target target"
-
-            # graphics tests requires --test-threads=1
-            ci-cargo +$toolchain test graphics_tests:: --manifest-path extendr-api/Cargo.toml --features tests-graphics $(if($target -ne 'default') {"--target=$target"} ) $env:EXTRA_ARGS '--' --nocapture --test-threads=1 -ActionName "Test extendr-api \w graphics for $target target"
-
-            ci-cargo +$toolchain test --manifest-path extendr-api/Cargo.toml --features tests-minimal $(if($target -ne 'default') {"--target=$target"} ) $env:EXTRA_ARGS '--' --nocapture -ActionName "Test extendr-api \w tests-minimal for $target target"
-            ci-cargo +$toolchain test test_result_list --manifest-path extendr-api/Cargo.toml --features tests-minimal,result_list $(if($target -ne 'default') {"--target=$target"} ) $env:EXTRA_ARGS '--' --nocapture -ActionName "Test extendr-api \w tests-minimal for $target target with `result_list` feature"
-            ci-cargo +$toolchain test test_result_condition --manifest-path extendr-api/Cargo.toml --features tests-minimal,result_condition $(if($target -ne 'default') {"--target=$target"} ) $env:EXTRA_ARGS '--' --nocapture -ActionName "Test extendr-api \w tests-minimal for $target target with `result_condition` feature"
-                      
-            ci-cargo +$toolchain test --manifest-path extendr-macros/Cargo.toml  $(if($target -ne 'default') {"--target=$target"} ) $env:EXTRA_ARGS '--' --nocapture -ActionName "Test extendr-macros for $target target"
-          
-          }
-        env:
-          BUILD_TARGETS: ${{ join(matrix.config.rust-targets, ',') }}
-          RUST_TOOLCHAIN: ${{ matrix.config.rust-version }}
-          EXTRA_ARGS: ${{ join(matrix.config.extra-args, ',') }}
```

Comment:
These tests are removed because we have removed the `use-bindgen` feature from libR-sys. We will need to reimplement this using an xtask per https://github.com/extendr/libR-sys/issues/189.

This needs to be removed so that we can address CRAN issues within the next 8 days. (Insane).